<launch>
  <!-- <param name="rx" type="double" value="0" />
  <param name="ry" type="double" value="-0" />
  <param name="rz" type="double" value="0" />
  <param name="tx" type="double" value="0" />
  <param name="ty" type="double" value="0" />
  <param name="tz" type="double" value="-0" /> -->
  <!-- <param name="data_path" type="string" value="/data/bs_hall/global_scans.pcd" /> -->
  
  <!-- <param name="data_path" type="string" value="/data/bs_hall/roi_scans_xyzi.pcd" /> -->
  <!-- <param name="data_path" type="string" value="/data/bs_hall/roi_scans_xyzrgb.pcd" /> -->
  <!-- <param name="data_path" type="string" value="/data/liyuan/spot0/fullview_recon/global_tf_cloud.pcd" /> -->

  <!-- Transform: ROI region(hall) -> global coarse(bs) -->
  <!-- <param name="rx" type="double" value="-3.11916" />
  <param name="ry" type="double" value="3.13113" />
  <param name="rz" type="double" value="2.27865" />
  <param name="tx" type="double" value="-13.19285" />
  <param name="ty" type="double" value="-23.6748" />
  <param name="tz" type="double" value="-0.110397" /> -->
  <!-- <param name="data_path" type="string" value="/data/bs_hall/spot0/fullview_recon/spot_rgb_cloud.pcd" /> -->

  <param name="rx" type="double" value="-0.0228948" />
  <param name="ry" type="double" value="0.027074" />
  <param name="rz" type="double" value="0.776748" />
  <param name="tx" type="double" value="-4.83285" />
  <param name="ty" type="double" value="-2.08588" />
  <param name="tz" type="double" value="-0.110397" />
  <!-- <param name="data_path" type="string" value="/data/bs_hall/spot0/fullview_recon/spot_cloud.pcd" /> -->
  <!-- <param name="data_path" type="string" value="/data/bs_hall/spot0/fullview_recon/global_registered_cloud.pcd" /> -->
  <!-- <param name="data_path" type="string" value="/data/bs_hall/spot0/fullview_recon/global_lio_registered_cloud.pcd" /> -->
  <param name="data_path" type="string" value="/data/bs_hall/spot0/fullview_recon/global_registered_rgb_cloud.pcd" />
  <!-- <param name="data_path" type="string" value="/data/rb1/spot0/fullview_recon/spot_cloud.pcd" /> -->

  <!-- <param name="rx" type="double" value="0.012864" />
  <param name="ry" type="double" value="0.001143" />
  <param name="rz" type="double" value="0.160874" />
  <param name="tx" type="double" value="0.299831" />
  <param name="ty" type="double" value="0.230617" />
  <param name="tz" type="double" value="-0.029438" /> -->
  
  <!-- <param name="data_path" type="string" value="/data/bs_hall/spot0/fullview_recon/global_registered_cloud.pcd" /> -->
  <!-- <param name="data_path" type="string" value="/data/liyuan/spot0/fullview_recon/global_lio_registered_cloud.pcd" /> -->
  
  <param name="rgb" type="bool" value="true" />
  <!-- <param name="rgb" type="bool" value="false" /> -->
  <param name="msg_size" type="int" value="10000" />
  <node name="rviz_pub" pkg="calibration" type="rviz_pub" output="screen">
  </node>
</launch>
